dom.bs

```diff
@@ -4071,35 +4041,30 @@ return <a>this</a>'s <a>shadow-including root</a> if
 
 <!-- TODO: domintro -->
 
-The <dfn attribute for=Node>nodeValue</dfn> attribute
-must return the following, depending on <a>this</a>:
+<p>The <dfn attribute for=Node>nodeValue</dfn> getter steps are to return the following, switching
+on <a>this</a>:
```

Comment:
The switching is still a bit weird. It might be best to do this as numbered steps so that you can say "this implements Attr / this implements CharacterData"

Reply:
Changing all instances of X node to that would be a rather massive undertaking and require changes in a lot more places. I was hoping that the new definition of node would avoid the need for doing that, but I wouldn't object to someone making that change.

Reply:
It's specifically the switches that are strange. E.g. in most programming languages if you do `switch (this) { case X: ... }` that compares `this === X`. But here you seem to be instead doing a `this instanceof X` comparison, which is not how switches usually work.

Reply:
Okay, your comment below made me think it was both. I'll look at reworking the switches tomorrow.

Reply:
Well, I think implements would be an improvement, but I can see not wanting to tackle that.